### PR TITLE
Add stub for WorkspaceConfig if workspace is created with Devfile

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Adds stub for WorkspaceConfig if workspace is created with Devfile

It is needed to make UI working somehow, otherwise, workspace created
with Devfile break UD and it's not even possible to remove such.

It should be reverted when clients will be able to handle workspace without workspace config properly.

### What issues does this PR fix or reference?
It's related to https://github.com/eclipse/che/issues/13079

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Comment by sleshchenko** [comment]
@ashumilova Could you provide your opinion about this PR, do you think we should merge it, or we can afford to wait until UD changes[1] will be merged? Do we have some estimation of them to be merged? Maybe we can separate some small piece to merge that won't cover all Devfile capabilities but would make UD stable if such workspace is created via API?
Sorry about so many questions here :smile_cat: 

[1] https://github.com/eclipse/che/pull/13198

**Review by garagatyi — APPROVED** [review]

**Review by ashumilova — APPROVED** [review]

**Comment by ashumilova** [comment]
I think better to merge your PR now.

**Comment by sleshchenko** [comment]
ci-test

**Comment by sleshchenko** [comment]
ci-test

**Comment by monaka** [comment]
Can I merge this? I found a test failure but I don't believe it is caused by this PR.

**Pull request merged**
